Source/Classes/PINRemoteImageManager.m

```diff
@@ -79,6 +81,16 @@ float dataTaskPriorityWithImageManagerPriority(PINRemoteImageManagerPriority pri
     }
 }
 
+// Reference: https://github.com/TextureGroup/Texture/blob/5dd5611/Source/Private/ASInternalHelpers.m#L60
+BOOL PINRemoteImageManagerSubclassOverridesSelector(Class subclass, SEL selector)
```

Comment:
We may should move this to a central place if there is one already.

Reply:
There is none that I'm aware of. Let's do it next time we have a similar need.